package.json

```diff
@@ -206,7 +211,7 @@
     "webpack": "^5.78.0"
   },
   "peerDependencies": {
-    "@appuniversum/ember-appuniversum": "^2.15.0",
+    "@appuniversum/ember-appuniversum": "^2.17.0 || ^3.0.0",
```

Comment:
We should support at least version 2.15.0 to be non-breaking.

Reply:
For that we would need to switch the path that the scss is imported from. `2.17.0` is the first version that supports the newer path. I don't know if we can use embroider macros in scss. I'll check.

Reply:
hmm okay interesting. This would technically mean that this causes a breaking release then :/

Reply:
But I guess this is not a problem, as we only need to import the ember-appuniversum styles in the consuming app, not in the addon. So depending on which version the host app has installed, they can choose between the two type of import paths. So I think we should just be able to set the minimum version to 2.15.0.

Reply:
Hmm, good point, I'll double-check that and set the minimum to 2.15.0 assuming it works.

Reply:
Minimum is now 2.15.0 again. I've kept the test app on 2.17.0 as there shouldn't be many differences and it means we can stick on the new import paths.

Reply:
Ok perfect!